services/gateway/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+JWT_SOURCE_64=$(base64 < /openssl-certs/madoc.pub);
+
+IFS=',' read -ra ADDR <<< "$JWT_SERVICES"
+for i in "${ADDR[@]}"; do
+    echo $(cat "./apps/$i.json" | jq --arg value "$JWT_SOURCE_64" '.jwt_source = $value') > "./apps/$i.json"
```

Comment:
An atomic move would be a better approach than doing the `cat` before the redirection kicks in (which immediately truncates the file).

Reply:
If you used an atomic move you could get rid of the `echo` and `cat`:

```
jq --arg {...} < "apps/$i.json" > "apps/$i.new.json"
if [ $? -eq 0 ]; mv "apps/$i.new.json" "apps/$i.json"; fi
```

Reply:
Ah good shout! I was having real problems trying to get JQ to read and write to the same file. This might solve the formatting too (currently new lines are just... lost)

Reply:
Yeah, if you redirect to a file with `>` the first thing bash does is truncates the file. So `cat $file > $file` produces an empty file.

Reply:
Fixed now 👍